portfolio/src/main/webapp/index.html

```diff
@@ -31,6 +31,41 @@ <h3>Zero Robotics</h3>
               <p><a href="http://zerorobotics.mit.edu/">Zero Robotics</a> is an international competition organized by MIT, NASA and ESA in which you have to program a miniaturized satellite abroad the International Space Station.</p>
           </div>
       </div>
+      <div id="hobbies" class="section">
+        <div class="title" onclick="toggleDescription('hobbies')">
+            <h2>Hobbies & Passions</h2>
+        </div>
+        <div class="description">
+            <div class="slider">
+                <div class="slide">
+                    <img src="images/tennis.jpg" alt="Tennis" class="slide-image">
+                    <div class="slide-text">I play tennis.</div>
+                </div>
+                <div class="slide hidden">
+                    <img src="images/AI.jpg" alt="Artificial Intelligence" class="slide-image">
+                    <div class="slide-text">
+                        I am fascinated by artificial intelligence and I would like to learn more about it.
+                    </div>
+                </div>
+                <div class="slide hidden">
+                    <img src="images/walk.jpg" alt="Walking" class="slide-image">
+                    <div class="slide-text">
+                        I like to walk or ride my bike in the fresh air.
+                    </div>
+                </div>
+                <div class="slide hidden">
+                    <img src="images/soccer.jpg" alt="Soccer" class="slide-image">
+                    <div class="slide-text">I am a soccer fan.</div>
+                </div>
+                <div class="slide hidden">
+                    <img src="images/tv.jpg" alt="TV Shows" class="slide-image">
+                    <div class="slide-text">I like watching TV Shows.</div>
+                </div>
+                <button class="change prev" onclick="previousSlide(this)">&#10094;</button>
+                <button class="change next" onclick="nextSlide(this)">&#10095;</button>
```

Comment:
There shouldn't be a need to use escaping for the special characters here. Just copy it in directly and make sure the file is saved in UTF-8: ❯
This helps with readability of the code and also makes it smaller.

Reply:
https://google.github.io/styleguide/htmlcssguide.html#Entity_References